project_national/national_baseline.yml

```diff
@@ -1,17 +1,18 @@
 schema_version: '0.3'
-os_version: 3.6.1
-os_sha: bb9481519e
+os_version: 3.5.1
```

Comment:
@prsh5175 Hmm, I don't think we need to be touching this file...? I think we can create the precomputed buildstock.csv file using `national_upgrades_gshp.yml`.

Reply:
Right. Forgot to undo/ignore this change.